sw/device/tock/boards/opentitan/earlgrey/peripherals/src/lib.rs

```diff
@@ -0,0 +1,11 @@
+//! Implementations for generic LowRISC peripherals.
+
+#![feature(asm, concat_idents, const_fn, core_intrinsics)]
+#![feature(in_band_lifetimes)]
+#![feature(exclusive_range_pattern)]
```

Comment:
Inconsistent pattern here, can this be made into a single `#![feature(...)]` or each broken out into distinct attributes?

Reply:
I think this probably should be done as a follow-up. At this point it has simply been copied from the tock repository of the revision that matches our existing downstream tock board implementation.